### DISCUSSION TITLE
Stopping notices

### DISCUSSION THREAD

**Discussion by lepo** [OP-TED/eForms-SDK#1026] [Q&A]
Hello,

We have some problems with stopping notices. Notices can only be stopped from beeing published while the state of the notice is SUBMITTED. 
If someone is submitting a notice on Friday at e.g. 14:30 and the EU publishing process takes place at 15:00, we / our customers only have 30 Minutes to stop the publication. Then the state of the notice is PUBLISHING from Friday to Monday morning. Inner this time range the customer has no chance to do anything with this notice.
There is also the same problem when submitting a notice very close to the start of the EU publishing process. Only a short time to react if a notice publication has to be stopped.
About the weekend topic. Why can't you start the Friday publication process on Monday morning or Sunday afternoon? 
For the other point you eventually can only take notices for the publishing process which are at least e.g. 3 hours submitted. The best way here is that we can configure this time in the xml for every notice. When this parameter is not defined the process is the same as now.

Best,
Günter

**Comment by YvesJo** [comment]
For this non-SDK related question, please get in touch with the Helpdesk.

**Comment by mdewinne** [comment]
You could use the "preferred publication date" as a workaround to achieve more or less the same result. E.g. set the PPD to Tuesday for notices created on Friday (afternoon). That way, you know they'll only get to status "Publishing" on Monday.

**Comment by ferrakaop** [comment]
Normally a valid notice that has been submitted is ready to be published on TED.  Stopping their publication is an exceptional case and there are periods when it's not possible. One of these periods is when the notice has been included in the afternoon export of the next day's OJS and it's in Publishing status.

The export process, done in the OP internal system, is done on working days with manual supervision, which is why we don't do it during the night or on weekends.

In the old pre-eForms systems, the publication date was fixed at 5 days after submission but there was still a cut-off moment in the afternoon when a notice couldn't be stopped any more (which was done manually, there was no API option like now).  

We now have the "as soon as possible" option, which reduces the time for "buyer's remorse", but as suggested by Michaël you can still choose your preferred publication date if you think you're likely to change your mind about a notice that was in theory ready to be published in the OJS.

But there will always be a period when the luxury of stopping publication is not available and we don't have any plans for a fully automatic process that would keep all options open until the last minute.
